### PR TITLE
feat: add tooltip to portfolio page

### PR DESCRIPTION
**Pull request by ClaraTschamon**
Issue: https://trello.com/c/LkiMD7uw/542-tooltip-portfolio-page-projectfacts-slice

### CODE REVIEW

File: src/components/portfolio/ProjectInfo/ProjectInfo.test.tsx [hunk 1]
```diff
@@ -66,6 +71,24 @@ describe('The ProjectInfo component', () => {
...
+      ...defaultProps,
+    });
+
+    const trigger = screen.getByText('Forecasted amount subtitle');
+    await userEvent.hover(trigger);
+
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent(
+      "This is the yearly project sink volume according to the TÜV-verified PDD, this value doesn't represent current availabilities."
+    );
```
Comment (lukasbals) [inline src/components/portfolio/ProjectInfo/ProjectInfo.test.tsx:85]:
```suggestion
    expect(await screen.findByRole('tooltip')).toHaveTextContent(
      messagesEn['xy']
    );
```

File: src/components/portfolio/ProjectInfo/ProjectInfo.test.tsx [hunk 2]
```diff
@@ -66,6 +71,24 @@ describe('The ProjectInfo component', () => {
     expect(screen.getByText('1 year')).toBeInTheDocument();
   });
 
+  it('sets the tooltip when hovered over the forecasted amount subtitle', async () => {
+    setup({
+      ...defaultProps,
+    });
+
+    const trigger = screen.getByText('Forecasted amount subtitle');
```
Comment (lukasbals) [inline src/components/portfolio/ProjectInfo/ProjectInfo.test.tsx:79]:
```suggestion
    const trigger = screen.getByText(messagesEn['xy']);
```

File: src/components/portfolio/ProjectInfo/ProjectInfo.tsx [hunk 3]
```diff
@@ -201,27 +202,34 @@ export const ProjectInfo: React.FC<ProjectInfoProps> = ({
...
-                  },
-                  {
-                    number: formatNumber(
-                      convertCo2AmountKgToTons(
-                        project.forecastedAmountYearly.toString()
+            <Tooltip
+              label={formatMessage({
+                id: 'features.projectInfo.properties.forecastedAmountYear.toolTip',
+              })}
+              role="trigger"
```
Comment (lukasbals) [inline src/components/portfolio/ProjectInfo/ProjectInfo.tsx:209]:
```suggestion
```

File: src/components/portfolio/ProjectInfo/messages.de.ts [hunk 4]
```diff
@@ -12,6 +12,8 @@ const messagesDe = {
     'SILVACONSULT® Forest Carbon Standard, ISO 14064-2',
   'features.projectInfo.properties.verificationStandard.value.MfKWCH':
     'Methodik für Klimaschutzprojekte im Wald für die Schweiz',
+  'features.projectInfo.properties.forecastedAmountYear.toolTip':
+    'Dies ist das jährliche Projekt-Sinkvolumen gemäß dem TÜV-geprüften PDD, dieser Wert stellt nicht die aktuellen Verfügbarkeiten dar.',
```
Comment (lukasbals) [inline src/components/portfolio/ProjectInfo/messages.de.ts:16]:
```suggestion
    'Dies ist das jährliche Projekt-Senkenvolumen gemäß dem TÜV-geprüften PDD, dieser Wert stellt nicht die aktuellen Verfügbarkeiten dar.',
```

File: src/components/portfolio/ProjectInfo/ProjectInfo.test.tsx (hunk 2 above)
Comment (ClaraTschamon) [reply src/components/portfolio/ProjectInfo/ProjectInfo.test.tsx:79]: This text was set in the test default Props and is not part of the messagesEn

**Review by lukasbals — APPROVED** [review]
Nice!

**Review by raulsteurer — APPROVED** [review]
🚀

**Review by lukasbals — APPROVED** [review]
Maybe a note on the commit message - I think better would be: 

`feat: add tooltip to project facts slice`

Reason: In the context of this repository (strapi-slices) you just updated a slice and it's no requirement to use it on a portfolio page.

**Pull request merged**